src/pages/s/[[...all]].tsx

```diff
@@ -56,17 +54,18 @@ export default function SearchIndex({initialSearchState, resultsState}) {
     createURL,
     onSearchStateChange,
   }
+
   return (
     <div>
+      <NextSeo noindex={!isEmpty(searchState.query)} title={pageTitle} />
```

Comment:
if the user has typed in a query we don't want to index anything. We only index when they have applied certain refinements and no query.